LICENSES/LicenseRef-Slint-Royalty-free-2.0.md

```diff
@@ -28,9 +28,11 @@ You may distribute the Software as part of an Application, modified or unmodifie
 
 ## 3. Limitations
 
-The license does not permit to distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
+The License does not permit to distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
 
-The license does not permit the use of the Software within Embedded Systems. An **Embedded System** is a computer system designed to perform a specific task within a larger mechanical or electrical system.
+The License does not permit the use of the Software within Embedded Systems. An **Embedded System** is a computer system designed to perform a specific task within a larger mechanical or electrical system.
+
+The License does not permit the distribution of Application that exposes the APIs, in part or in total, of the Software.
```

Comment:
This is now much better than previous attempts, although still handwavy with the word "exposes", I think the intention is understandable, thought about making suggestion like this too.
As an another alternative was thinking of defining "Applications that are expected to be used as buildings blocks of other Applications" but that prohibits usecases like game engines, and would be even harder to define correctly, and probably is not the direction you want to go to.
Good job with listening to the feedback.

Reply:
Ah, also another another way was to restrict Slint uses for applications that explicitly advertise themselves as GUI toolkits.

Reply:
@kanashimia Thanks for the feedback. We want to keep the text as simple as possible. If the intention is understandable (like you mentioned) then its already good. We can always clarify on a case-by-case basis.